server/apiCalls/transformers/memosCourseOfferings.test.js

```diff
@@ -37,10 +43,16 @@ const offering_SF1625_202122 = {
   courseCode: 'SF1625',
   offeringId: '2',
```

Comment:
All `ladokRoundIds`, `offeringId`, `roundId`, `roundList`, `round` and so on has to be removed or replaced to be sure tests working properly. Otherwise it is unclear why tests working, because it has old values or because it has new values

Reply:
Removed